docs/core/tools/project-json-to-csproj.md

```diff
@@ -265,23 +300,6 @@ dependency version of a project reference.
 </ItemGroup>
 ```
 
-#### type: platform
-```json
-{
-  "dependencies": {
-    "Microsoft.NETCore.App": {
-      "version": "1.1.0",
-      "type": "platform"
```

Comment:
This was the bit that @blackdwarf judged inaccurate and said it was best to wait to add info re. type:platform. He told to wait until we have a good way to explain this.